test/dst/dst.go

```diff
@@ -490,8 +497,9 @@ func (d *DST) Model() porcupine.Model {
 						<td align="right">%d</td>
 						<td align="right">%d</td>
 						<td align="right">%d</td>
+						<td align="right">%s</td>
 					</tr>
-				`, t.value.Id, t.value.State, t.value.RootPromiseId, t.value.ExpiresAt, t.value.Timeout, *t.value.CreatedOn)
+				`, t.value.Id, t.value.State, t.value.RootPromiseId, t.value.ExpiresAt, t.value.Timeout, *t.value.CreatedOn, completedOn)
```

Comment:
Not super important but we have a utility function for this purpose
```suggestion
				`, t.value.Id, t.value.State, t.value.RootPromiseId, t.value.ExpiresAt, t.value.Timeout, *t.value.CreatedOn, util.SafeDeref(t.value.CompletedOn))
```

I also think the indenting is off a little on this line.